antarest/launcher/web.py

```diff
@@ -120,4 +120,25 @@ def get_versions(
         logger.info(f"Fetching version list")
         return service.get_versions(params=params)
 
+    @bp.post(
+        "/launcher/jobs/{job_id}/kill",
+        tags=[APITag.launcher],
+        summary="Kill job",
+    )
+    def kill_job(
+        job_id: str,
+        engine: Optional[str] = None,
```

Comment:
you don't need this, when a job is created, the launcher used is store in the job_result object